torch/csrc/Module.cpp

```diff
@@ -614,6 +628,7 @@ static PyMethodDef TorchMethods[] = {
   {"_is_xnnpack_enabled", (PyCFunction)THPModule_isEnabledXNNPACK, METH_NOARGS, nullptr},
   {"_is_torch_function_enabled", (PyCFunction)THPModule_isEnabledTorchFunction, METH_NOARGS, nullptr},
   {"_disabled_torch_function_impl", (PyCFunction)THPModule_disable_torch_function, METH_VARARGS, nullptr},
+  {"Assert", (PyCFunction)THPModule_Assert, METH_VARARGS, nullptr},
```

Comment:
why... capitalized?

Reply:
oh because assert is a reserved keyword

Reply:
`assert` is reserved in Python, just keeping the naming consistent throughout.  There is a TODO(before land) to align on best actual name to use.